### ISSUE TITLE
Support for Juniper

### ISSUE THREAD

**Issue by ghost** [Wyko/network-scraper#14]
Please let me know if you'd like to incorporate support for JUNOS devices in the future, I would be happy to help out.

**Comment by Wyko** [comment]
Absolutely! Please create a new device class inheriting `devices.base.network_device` and add the appropriate logic. 

Also, I've only barely started adding `pytest` tests to this project, but if there are any relevant tests you can add to the device, please feel free to add them as well.

**Comment by Wyko** [comment]
Any progress?
